### DISCUSSION TITLE
Web page with Astroid doesn't display

### DISCUSSION THREAD

**Discussion by AntonioSco** [templaza/astroid-framework#895] [Q&A]
Hi,
I just installed Astroid extension in J 4.4.9, Windows 10.
Previoulsy my webpage had a layout based on the Cassiopea template. I made a copy of Astroid and made it my default template and changed some of the settings. However when I display the page it still shows the earlier Cassiopea layout. Is there anything I am missing here?

The instructions on how to "Creating a basic Astroid Template" say that the original template must be unistalled and the Astroid Template_zero  file deleted. Could this be the reason why I am not seeing the changes displayed? (Why if so?) On the other hand J4 doesn't let me unistall the default Astroid template.

**Comment by Banuk1616** [comment]
I don't quite understand what they did exactly. This may be due to my poor English. 
What exactly did you install?
You need to install the framework https://github.com/templaza/astroid-framework/releases/download/v3.1.8/astroid-framework-3.1.8.zip and one of the templates (zero, one or two). 

Best regards

**Comment by AntonioSco** [comment]
I installed exactly 3.1.8 but nothing happened, not even after clearing the cache (both the J4 and Firefox)
I uninstalled both the framework and the Astroid 2 template, and reinstalled them but again, I can only see the cassiopea template layout shown.
Even after disabling Cassiopea, initially the layout being used was still Cassiopea's
I've tried other templates as well, with the same result while Cassiopea was enabled but not set as the default template.
After a while though, after Cassiopea was disabled, the other templates would load, as expected when set as default.
I cleared the cache several times (both the J4's and firefox's), and I wonder if that has had any effect at all.

Has anyone experienced this behavior? I wasted quite a lot of time figuring out why this was happening...
